diplomacy/web/src/gui/forms/message_form.jsx

```diff
@@ -341,17 +356,19 @@ static countries = [
                             possibleSplit[i] = "moving to";
                         }
                     }
+
                     let supportString = "";
                     possibleSplit.forEach(element => {
                         supportString += element + " ";
                     });
-                    if (supportString !== ""){
+
+                    if (supportString) {
                         supportMoves[locKey] = supportString.trim();
                     }
                 }
             });
             return (Object.entries(supportMoves).map(([key, value]) => {
-                    return(
+                    return (
```

Comment:
If you'd like ignore white-space changes while reviewing, since I know I did small drive-by style changes:

https://github.com/jataware/diplomacy/pull/25/files?w=1